pallets/msa/src/rpc/src/tests/mod.rs

```diff
@@ -0,0 +1,178 @@
+mod rpc_mock;
+
+use super::*;
+use rpc_mock::*;
+
+use common_primitives::node::BlockNumber;
+use pallet_msa_runtime_api::MsaRuntimeApi;
+use std::{sync::Arc, vec};
+use substrate_test_runtime_client::runtime::{AccountId, Block};
+
+const PROVIDER_WITH_DELEGATE_A: ProviderId = ProviderId(1);
+const DELEGATE_A: DelegatorId = DelegatorId(2);
+const DELEGATE_B: DelegatorId = DelegatorId(3);
+const PROVIDER_WITH_DELEGATE_A_AND_B: ProviderId = ProviderId(4);
+const SCHEMA_FOR_A: u16 = 1;
+const SCHEMA_FOR_A_AND_B: u16 = 2;
+const SCHEMA_FOR_B: u16 = 3;
+const NOT_EXIST_MSA: u64 = 100;
+
+sp_api::mock_impl_runtime_apis! {
```

Comment:
This is the runtime mock. So we are only testing the code in the rpc side of things.